src/main/java/hw4/pages/HomePage.java

```diff
@@ -0,0 +1,48 @@
+package hw4.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends AbstractPage {
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    private static final String URL = "https://jdi-testing.github.io/jdi-light/index.html";
+
+    @FindBy(id = "user-icon")
+    private WebElement userIcon;
+
+    @FindBy(id = "name")
+    private WebElement userNameTextField;
+
+    @FindBy(id = "password")
+    private WebElement passwordTextField;
+
+    @FindBy(id = "login-button")
+    private WebElement loginButton;
+
+    @FindBy(id = "user-name")
+    private WebElement userNameText;
+
+    public void open() {
+        driver.get(URL);
+    }
```

Comment:
What is the class component ordering which described in the Java Code Convention?
Please check other classes?